strapi/strapi
# enhancement!: create should send 201 status code

### What does it do?

All create methods throughout strapi now use ctx.created, most importantly the content api. In cases I wasn't sure it was safe to use ctx.created I explicitly set status to 201

### Why is it needed?

Create actions should result in a 201 http status code

### How to test it?

All the updated tests test it pretty thoroughly.

But to test manually, the default status for a create action in a custom controller should be 201 so:

- create a content type
- use the content API to create an item of that content type
- the response status code should be 201
- ensure that update/delete/etc actions are all still 200 status

Admin API can essentially be tested the same way, just through different endpoints.

### Related issue(s)/PR(s)

DX-1118
